app/views/va_facilities/_facility_directory_combo_box.html.erb

```diff
@@ -0,0 +1,14 @@
+<label for="<%= facility_selector %>" class="usa-label line-height-26">Facilities</label>
+<div class="usa-combo-box margin-top-1 margin-bottom-2 desktop:margin-y-0">
+  <select id="<%= facility_selector %>" class="usa-select" name="<%= facility_selector %>">
+    <% @facilities.each do |va| %>
+      <option value=<%= va["id"] %>
+          <% if params[:facility].present? && va["id"] == params[:facility].to_i%>
+            selected
+          <% end %>
+      >
+        <%= va["official_station_name"].to_s + ' (' + va["common_name"].to_s + ')' %>
```

Comment:
We shouldn't need the `to_s` because they should already be strings in the DB. We should also interpolate here